common/utils_win.h

```diff
@@ -18,14 +18,24 @@ namespace internal {
 // Google Chrome.
 const DWORD kBufferSize = 4096;
 
+// Named pipe prefixes used for agent and client side of pipe.
+constexpr char kPipePrefixForAgent[] = "\\\\.\\pipe\\";
```

Comment:
nit: Maybe it's clearer to use a R"()" string to improve readability?

Reply:
Done.